app/steps/ui/shutterpage/index.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const Step = require('app/core/steps/Step');
+
+module.exports = class ShutterPage extends Step {
```

Comment:
I'd prefer to have the `module.exports` at the end of the file.

See `app/steps/ui/deceased/deathcertificate/index.js` for an example.